docs/conf.py

```diff
@@ -184,7 +217,7 @@
 # modindex_common_prefix = []
 
 # If true, keep warnings as "system message" paragraphs in the built documents.
-# keep_warnings = False
+keep_warnings = True
```

Comment:
This is only information for the developer. Not everyone knows this switch. The end-user will not see these messages because we have a zero error policy in the documentation.  Console warnings continue to appear, so this option is safe.